### PR TITLE
DFPL-622: Remove CMO from MA21C50113

### PR DESCRIPTION
**Pull request by AlistairEdwinOsborne**
https://tools.hmcts.net/jira/browse/DFPL-622

Add migration script to remove draft CMO from MA21C50113

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by qasimnawaz-moj — APPROVED** [review]
lgtm

**Review by DanCatchpole — APPROVED** [review]
approving after merge conflict was resolved!

**Pull request merged**
